alembic/ddl/impl.py

```diff
@@ -136,6 +139,32 @@ def static_output(self, text: str) -> None:
         self.output_buffer.write(text + "\n\n")
         self.output_buffer.flush()
 
+    def version_table_impl(
```

Comment:
I think we should make this more future proof by doing all keyword args with room for expansion.  these are keyword only arguments:

```
def version_table_impl(
   self, *, version_table, version_table_schema, version_table_pk, **kw
)
```

Reply:
Fixed